ui/package.json

```diff
@@ -14,10 +14,36 @@
     "react-scripts": "1.1.1"
   },
   "scripts": {
+    "lint": "prettier **/*.js **/*.css",
+    "fix": "prettier --write **/*.js **/*.css",
     "precommit": "pretty-quick --staged",
     "start": "PORT=4400 react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
+    "test": "jest",
```

Comment:
Wasn't able to get `npm test` working. (Sent you details offline.)

Reply:
This is an involved change that would likely involve replacing our linter with a less opinionated and more configurable option. If you're adamant about this, we can create an issue for it and talk about prioritizing it. However, it's out of scope for this PR.